content/practices/event-storming-process-modelling/index.md

```diff
@@ -0,0 +1,151 @@
+         - The Audience listens to the narrator and assists in discovering the process
+         - A Scribe records the process on the board
+       - Any participant can take on any of these roles and it should be encouraged for roles to rotate
+
+    - Questions: If a question cannot be answered straight away then leave it on the board and return to it later.
+    - Grammar: Add grammar appropriately but do not get bogged down by it
+      1. Events: These represent events that are "Interesting" to the wider system. When understanding what is "interesting", you might refer to the description given by Martin Fowler in his article [What do you mean by “Event-Driven”?](https://martinfowler.com/articles/201701-event-driven.html)
+      1. Policies: This sticky represents a business decision to do something, sometimes this is written as a short sentence or can be labeled and described somewhere else. Any description should include the `event` they are listening to an `actor` if one is required, a `business decision` to be made and an `action` to take
+      1. Actors: This sticky represents a human interaction and should be placed with policies to identify who is responsible of making the decision and performing the task, a policy without an actor is automated
+      1. Action: An action that must be performed due to policy and produces one or more interesting events
+      1. Read Models: Data required to make decisions or perform commands
+      1. System: external/3rd party tools such as; email, excel, notepad, or any required COTS application
+      Note: it does not always need to happen in this order and more grammar may be added, so it is best to leave enough space or be prepared to shift post-its further to the right as you make changes.
+    - Negative/Alternative Outcomes: If a command can result in multiple or alternative events then all events should be placed vertically
+    - Pain Points and Opportunities: Place a sticky to represent any pain points and any opportunities that are identified at the time. Pain points can be around the process itself or the current system or tools supporting the process
+
+1. Narrate the process again; correct inaccuracies and add missing grammar. You can also encourage the discovery of negative events
+   Repeat this until you are satisfied you have enough of the process to identify a thin slice.
+
+1. Review Questions and Pain Points: Try to answer as many questions as possible and rationalize pain points and opportunities
+
+#### Success/Expected Outcomes
+   - Capture information to assist further in the development process
+     - Ubiquitous Language
+     - Domain Events
+     - Policies for Decision Making 
+     - Actors
+     - Read Models
+     - Manual Actions + System Commands
+     - Integrations with tools and apps
+     - Pain Points + Opportunities
+   - Identify one or more thin slices. Try to identify a simplified end-to-end process that can provide the most value. 
```

Comment:
It would be good to define the term as you introduce it. The definition is a the end of the article.